types/analytics-node/index.d.ts

```diff
@@ -33,9 +33,11 @@ declare namespace AnalyticsNode {
   }
 
   interface Integrations {
-    [index: string]: boolean;
+    [integration_name: string]: IntegrationValue
   }
 
+  type IntegrationValue = boolean | { [integration_key: string] : any };
```

Comment:
Looks like linting is turned off. Can you please remove the list of tslint rule exceptions in `tslint.json` and update this package appropriately? Should be reasonable as it's fairly small.